pandas/core/indexes/interval.py

```diff
@@ -198,11 +202,11 @@ class IntervalIndex(IntervalMixin, Index):
 
     See Also
     --------
-    Index : The base pandas Index type
-    Interval : A bounded slice-like interval; the elements of an IntervalIndex
-    interval_range : Function to create a fixed frequency IntervalIndex
-    cut, qcut : Convert arrays of continuous data into Categoricals/Series of
-                Intervals
+    Index : The base pandas Index type.
+    Interval : A bounded slice-like interval; the elements of an IntervalIndex.
+    qcut : Quantile-based discretization function.
+    cut : Return indices of half-open bins to which each value of x belongs.
```

Comment:
I know this is the explanation in the cut docstring, but it is actually not correct (we are having a discussion about this in the PR doing the cut function :)). I would keep the explanation that was there before about converting array of continuous data into intervals